site_scons/buildutils.py

```diff
@@ -1533,27 +1534,29 @@ def check_module(name):
         msg.extend(line for line in info if not line.startswith("versions:"))
         logger.warning("\n| ".join(msg))
 
+    if env["python_package"] == "y":
+        logger_method = logger.error
+        exit_on_error = True
+    else:
+        logger_method = logger.warning
+        exit_on_error = False
 
-    python_version = versions["python"]
-    if python_version < env["python_min_version"]:
-        logger.error(
+    python_version = versions.get("python")
+    if not python_version or python_version < env["python_min_version"]:
```

Comment:
Shouldn’t this be `python_version is not None`?

Reply:
No, this is correct. If `python_version` isn't in the dictionary and `.get()` returns None, we want to warn/error and exit if appropriate